.kokoro/dashboard.sh

```diff
@@ -20,9 +20,40 @@ scriptDir=$(realpath $(dirname "${BASH_SOURCE[0]}"))
 ## cd to the parent directory, i.e. the root of the git repo
 cd ${scriptDir}/..
 
+outputFile="$scriptDir/../dashboard/target/tmp/output.txt"
 ## Move into the dashboard directory
 cd dashboard/
 
 echo -e "\n******************** BUILDING THE DASHBOARD ********************"
 
-mvn --fail-at-end clean install
+mvn --fail-at-end -DskipTests=true clean install
+INSTALL_RETURN_CODE=$?
```

Comment:
I don't think these are working as you intend them to -- `set -e` at the top of the file makes the script terminate if any command fails.

Reply:
nit: If the compile/install fails, is it even worth continuing?